src/_langs/en/tools/setup/workspace/setup-snippets.markdown

```diff
@@ -0,0 +1,172 @@
+---
+layout: tools-article
+title: "Set Up Snippets"
+rss: false
+seotitle: "Set Up Snippets in Chrome DevTools"
+description: "Use DevTools snippets to save small scripts, bookmarklets, 
+and utilities so that they're always available to you while debugging in the
+browser."
+introduction: "Use DevTools snippets to save small scripts, bookmarklets, 
+and utilities so that they're always available to you while debugging in the
+browser."
+authors:
+  - kaycebasques
+article:
+  written_on: 2015-10-01
+  updated_on: 2015-10-01
+  order: 1
+collection: workspace
+takeaways:
+    - Use snippets to run small scripts from any page.
+    - Run portions of snippets in the console with the "Evaluate in
+      Console" feature.
+    - Map snippets to external files with workspaces.
+    - Remember that popular features from the Sources panel, like
+      breakpoints, also work with snippets.
+---
+
+{% wrap content %}
+
+Snippets are small scripts that you can author and execute within the 
+**Sources** panel of Chrome DevTools. You can access and run them from 
+any page. When you run a snippet, it executes from the
+context of the currently open page. 
+
+If you have small utilities or debugging scripts which you find yourself 
+using repeatedly on multiple pages, consider saving the scripts as snippets. 
+You can also use snippets as an alternative to 
+[bookmarklets](https://en.wikipedia.org/wiki/Bookmarklet).
+
+{% include modules/toc.liquid %}
+
+{% include modules/takeaway.liquid list=page.takeaways %}
+
+## Access snippets
+
+Use the snippets panel in Chrome DevTools to create, modify, run, 
+and delete snippets.
+
+1. [Open DevTools](setup-devtools#access-devtools) and go to the 
+   **Sources** panel.
+
+1. Click the layout toggle in the top-left corner of DevTools.
+
+1. Click on the **Snippets** panel.
+
+The video below shows how to navigate to the snippets panel. The **Sources**
+tab is clicked, then the layout toggle in the top-left corner is clicked, 
+then the **Snippets** panel is clicked.
+
+{% animation animations/access-snippets.mp4 %}
+
+## Create snippets
+
+The left side of the snippets panel displays the list of snippets available
+on your machine. The right panel displays an editor which you can use to
+create and modify snippets. Once you have created snippets, clicking on
+a snippet filename in the left panel makes DevTools display the 
+snippet's source code in the right panel.
+
+1. Right-click inside the file list panel.
+
+1. Select **New** to create a new snippet file.
+
+1. Type in a filename for your snippet, or type `Enter` to use the
```

Comment:
type "Enter" reads like I should type the word Enter somewhere :) Maybe "press Enter"? Just a nit though, feel free to ignore.